content/en/dashboards/sharing.md

```diff
@@ -19,7 +19,39 @@ further_reading:
 
 ## Overview
 
-Shared graphs and screenboards allow you to display metric, trace, and log visualizations outside of Datadog.
+Shared dashboards and graphs allow you to display metric, trace, and log visualizations outside of Datadog.
+
+## Dashboards
+
+**Share an entire dashboard publicly by generating a URL**:
+1. On the dashboard's page, click the settings cog in the upper right.
+2. Choose the **Generate public URL** option.
+3. Choose configuration options under "Time & Variable Setings" for "Allow changing timeframe" and/or template variables visible tags.
+4. Copy the URL and click **Done**.
+
+**Share with individual emails**: 
+1. On the dashboard's page, click the settings cog in the upper right.
+2. Choose the **Generate public URL** option.
+3. Select "Only specified people" can access this dashboard.
+4. Input the emails you would like to share your dashboard with.
+5. Choose configuration options under "Time & Variable Setings" for "Allow changing timeframe" and/or template variables visible tags.
```

Comment:
```suggestion
5. Under **Time & Variable Settings**, configure your desired options for the time frame and whether users can change it; which tags are visible for selectable template variables; and which of those template variable values is the default on the shared dashboard page.
```

Reply:
`Under **Time & Variable Settings**, configure your desired options for the time frame and whether users can change it; which tags are visible for selectable template variables.` 

Let's leave off the second half, it doesn't make sense to include